internal/controller/cilium_flow_collector.go

```diff
@@ -4,6 +4,7 @@ package controller
 
 import (
 	"context"
+	cryptoTls "crypto/tls"
```

Comment:
No capital letters in package names/aliases.
```suggestion
	cryptotls "crypto/tls"
```